### ISSUE TITLE
I can't export the point cloud file，Progress hasn't budged, but gup takes 90%

### ISSUE THREAD

**Issue by cj-Noah** [nerfstudio-project/nerfstudio#3615]
(nerf) D:\Pycharm\Nerfstudio>ns-export pointcloud --load-config outputs\poster\nerfacto\2025-03-17_172548\config.yml --output-dir exports/pcd/ --num-points 30000 --remove-outliers True --normal-method open3d --save-world-frame False --obb_center 0.0000000000 0.0000000000 0.0000000000 --obb_rotation 0.0000000000 0.0000000000 0.0000000000 --obb_scale 1.0000000000 1.0000000000 1.0000000000
D:\Pycharm\Nerfstudio\nerfstudio\utils\misc.py:183: RuntimeWarning: Windows does not yet support torch.compile and the performance will be affected.
  warnings.warn(
[19:08:03] Auto image downscale factor of 2                                                 nerfstudio_dataparser.py:484
Variable resolution, using variable_res_collate
Loading latest checkpoint from load_dir
✅ Done loading checkpoint from outputs\poster\nerfacto\2025-03-17_172548\nerfstudio_models\step-000029999.ckpt
D:\Pycharm\Nerfstudio\nerfstudio\data\scene_box.py:105: UserWarning: To copy construct from a tensor, it is recommended
to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than
torch.tensor(sourceTensor).
  comp_l = torch.tensor(-S / 2)
D:\Pycharm\Nerfstudio\nerfstudio\data\scene_box.py:106: UserWarning: To copy construct from a tensor, it is recommended
to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than
torch.tensor(sourceTensor).
  comp_m = torch.tensor(S / 2)
☁ Computing Point Cloud ☁ ━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━  55% --:--

![Image](https://github.com/user-attachments/assets/71d66dc7-4db4-4e57-9f7e-8d2d0c62a138)

**Comment by SirErico** [comment]
Out of curiosity, have you managed to solve this issue?

**Comment by mariapatni** [comment]
I am also having this issue

**Comment by SirErico** [comment]
I think the `--obb_scale` flag is very resource intensive. Removing it from the command solved the problem for me.
